pkg/cloud/services/compute/instance_types.go

```diff
@@ -30,7 +30,7 @@ import (
 
 // InstanceSpec defines the fields which can be set on a new OpenStack instance.
 //
-// InstanceSpec does not contain all of the fields of infrav1.Instance, as not
+// InstanceSpec does not contain all of the fields of infrav1.Spec, as not
```

Comment:
This doesn't look correct.

Also, this comment was relevant context when I originally forked this struct from Instance. I think we can just delete it now.